### DISCUSSION TITLE
Docker containerization

### DISCUSSION THREAD

**Discussion by Ymir-badam** [neokd/NeoGPT#25] [Ideas]
Hey @neokd was thinking about to containerize the project so that it would make easy for new contributors and users to access and run it on their local machine

**Comment by neokd** [comment]
Great idea @Ymir-badam we can implement it.

**Comment by Ymir-badam** [comment]
hey @neokd I am working on this will give you update until the next week just taking a break on this weekend I hope u enjoy u r weekend tooo.

**Comment by neokd** [comment]
@Ymir-badam I have created an issue for adding docker support
